File: Api_Recordis/src/main/java/com/recordis/api_recordis/model/Actividad.java
package com.recordis.api_recordis.model;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@Document(collection = "actividades")
public class Actividad {

    @Id
    private String id;

    private String titulo;
    private String descripcion;
    private String hora;
    private boolean completada;
    private String usuarioId;
    private String fecha;
}

File: recordis/domain/entities/Actividad.js
// domain/entities/Actividad.js
export const crearEntidadActividad = (raw) => ({
  id: raw.id ?? raw._id ?? raw.idActividad,

  titulo: raw.titulo,
  descripcion: raw.descripcion ?? '',
  hora: raw.hora,
  completada: raw.completada ?? false,
  usuarioId: raw.usuarioId,
  fecha: raw.fecha,

  esCitaSalud: raw.esCitaSalud ?? false,
});
